src/presentational/Classroom.jsx

```diff
@@ -70,6 +71,30 @@ export default class Classroom extends Component {
       </div>
     )
   }
+  
+  renderAssignmentsList() {
```

Comment:
assuming we have `defaultProps` properly set:

renderAssignmentsList() {
  return (
    <table className="table table-hover">
      <thead>
        <tr>
          <th>Name</th>
          <th>Actions</th>
        </tr>
      </thead>
      <tbody>
        { this.props.assignments.data.map(assignment => 
          <tr key={`assignment_${assignment.id}`}>
            <td>{assignment.attributes.name}</td>
            <td><button>EDIT</button></td>
          </tr>
        ) }
      </tbody>
    </table>
  );
}